LDM-592.tex

```diff
@@ -1257,36 +1257,34 @@ \section{Glossary}
 
 \item[Composite Dataset] A Dataset made up of more than one items which are themselves Datasets (e.g.\ \texttt{MaskedImage} + PSF + Metadata = \texttt{Exposure})
 
-\item[DataGraph] A graph structure containing DatasetRefs and DataUnits as vertices and the connections between them as edges.
+\item[Quantum] An indivisible unit of work that consumes and produces Datasets.
+
+\item[QuantumGraph] A graph structure containing DatasetRefs and Quanta as vertices and the connections between them as edges.
 
 \item[Dataset] Represents a single discrete entity of data, with associated metadata (e.g.\ a particular \texttt{calexp} for a particular instrument corresponding to a particular visit and sensor).
 
-\item[DatasetExpression] A expression matching one or more Datasets; something like a SQL WHERE against a common schema in which each DataUnit is a table.
+\item[DatasetExpression] A expression matching one or more Datasets; something like a SQL WHERE against a common schema in which each Dimension is a table.
 
-\item[DatasetIdentifier (or DataID)] A tuple of DataUnits that can be used to label a Dataset.
+\item[DataCoordinate (or DataID)] A mapping with Dimension keys and identifiers for those dimension that can be used to label a Dataset.
 
-\item[DatasetIdentifierType] A tuple of DataUnitTypes that describe the label for a DatasetType.
+\item[DimensionGraph] A tuple of Dimensions that describe the label for a DatasetType.
 
-\item[DatasetRef] The combination of a DatasetIdentifier and the name of DatasetType, which can be used to uniquely identify a Dataset within a DataRepository.
+\item[DatasetRef] The combination of a DataCoordinate and the name of DatasetType, which can be used to uniquely identify a Dataset within a DataRepository.
 
-\item[DatasetType] The conceptual type of which Datasets are instances (e.g.\ \texttt{calexp}).  A new DatasetType can be defined by combining a StorageClass, a DatasetIdentifierType, and a string name.
+\item[DatasetType] The conceptual type of which Datasets are instances (e.g.\ \texttt{calexp}).  A new DatasetType can be defined by combining a StorageClass, a DimensionGraph, and a string name.
 
-\item[DataRepository] A collection of Datasets that has the following three properties:
+\item[Collection] A group of Datasets that has the following three properties:
 \begin{itemize}
 \item Has at most one Dataset per DatasetRef;
 \item Has a label that humans can parse (i.e.\ RepositoryRef);
```

Comment:
Probably `RepositoryRef` should be defined - and is this really what's meant here?